ironfish/src/fileStores/config.ts

```diff
@@ -32,6 +32,7 @@ export type ConfigOptions = {
    */
   p2pStunServers: string[]
   databaseMigrate: boolean
+  disableWalletSync: boolean
```

Comment:
Most config flags here are prefixed with `enable`. Can this be consistent and default to a different value?

Reply:
Cool, I like that more. I'll default it to true